src/main/webapp/resources/js/pages/projects/samples/components/FilterByFileModal.jsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { Col, Form, Input, List, Modal, Row, Typography } from "antd";
+import { useSelector } from "react-redux";
+import { CheckCircleTwoTone, WarningTwoTone } from "@ant-design/icons";
+import { green6, red6 } from "../../../../styles/colors";
+import VirtualList from "rc-virtual-list";
+import { SPACE_SM } from "../../../../styles/spacing";
+
+const ROW_HEIGHT = 43;
+
+/**
+ * React component to render a modal to allow the user to select a file containing coma or new
```

Comment:
Typo in `coma` and `sample name` should be `sample names`

Reply:
Updated in 1d244ed0c434997c30f03d3c564e941ee05872ed

Reply:
I think an accidentally `'` was added to the end of `the samples'`

Reply:
Nope, that's actually proper grammar I believe

Reply:
I don't believe it is :)

Reply:
Guess you were right! :)